orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -324,6 +325,21 @@ def _get_exported_model(self, input_schema: _ModelInputOutputSchemaType, *inputs
         # FlattenedModule needs _InputInfo to expand user input from *args to *args + **kwargs
         self._flattened_module._input_info = self._input_info
 
+        # Leverage cached model if available
+        cache_dir = self._runtime_options.ortmodule_cache_dir
+        rank = torch.distributed.get_rank() if torch.distributed.is_initialized() else 0
```

Comment:
Can use the utility now:

https://github.com/microsoft/onnxruntime/blob/eeef15788804b0217d46ad6d1c2744585ad98d0c/orttraining/orttraining/python/training/ortmodule/_utils.py#L449-L454